lib/cdo/asset_helper.rb

```diff
@@ -40,8 +49,20 @@ def webpack_asset_path(asset)
     raise "Invalid webpack asset name: '#{asset}'" unless path
     path
   end
+
+  # Returns the asset path for assets handled by the offline feature's webpack build.
+  def webpack_offline_asset_path(asset)
```

Comment:
one idea to keep in mind is that if we start to add more service workers for things other than offline, it might make sense to call this something like `webpack_service_worker_asset_path`. we can decide that whenever we add a next service worker.